runtime/lua/vim/lsp/handlers.lua

```diff
@@ -384,6 +384,12 @@ local make_call_hierarchy_handler = function(direction)
   end
 end
 
+--see https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_foldingRange
+M['textDocument/foldingRange'] = function(_, _, result, _, bufnr, _)
```

Comment:
This is out of date with the new handler signature.

Reply:
Should be fixed now.